src/app/ngst/table/table.component.ts

```diff
@@ -141,12 +154,18 @@ export class TableComponent implements OnInit, OnChanges, AfterViewInit {
   }
 
   changeRow(row: any, column: Column, newValue: any) {
+    //Flag for change flow on update table
+    this.changeRowValue = true;
     // Clone the user's data
     const clone = Object.assign({}, ...row);
 
     // Mutate the clone
     column.editor.edit(clone, column, newValue);
 
+    const rowFound = this.rawDataSource.data.findIndex(x => x.uuid == clone.uuid);
```

Comment:
`NGST` cannot know that row items have a `uuid` - this is a generic table.

If this sorting persistence requires knowledge of a row ID (and I think it does) then I think a wrapper for `ngst` will be required in the parent app. The sort order would need to be maintained there.

Reply:
I changed 
```suggestion
    const rowFound = this.rawDataSource.data.findIndex(x => x.uuid == clone.uuid);
```
by find row objects without use any properties

```suggestion
    const rowFound = this.rawDataSource.data.indexOf(row);
```